cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/configurations/UpdateStatusConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package com.farao_community.farao.cse.import_runner.app.configurations;
+
+import com.farao_community.farao.cse.import_runner.app.util.UpdateStatusProducer;
+import com.farao_community.farao.gridcapa.task_manager.api.TaskStatusUpdate;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.Message;
+import reactor.core.publisher.Sinks;
+
+@Configuration
+public class UpdateStatusConfiguration {
```

Comment:
I don't really see the advantage of using that instead of StreamBridge object

Reply:
streamBridge has returned. I removed it because i thought it was not working any more but i removed his configuration